test/e2e/scenario/rum/eventBridge.scenario.ts

```diff
@@ -60,4 +60,23 @@ describe('bridge present', () => {
       expect(serverEvents.rumViews.length).toEqual(0)
       expect(bridgeEvents.rumViews.length).toBeGreaterThan(0)
     })
+
+  createTest('forward internal monitoring to the bridge')
```

Comment:
that is not RUM specific, should it be in internal monitoring scenario instead?

Reply:
Indeed

Reply:
both approaches are fine to me